subprojects/version-info/version-info.gradle

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+task buildReceiptResource(type: Copy, dependsOn: rootProject.createBuildReceipt) {
+    into "$generatedResourcesDir/org/gradle"
+    from rootProject.createBuildReceipt.receiptFile
```

Comment:
We could make the `sources.main.resources` generated by the appropriate receipt task and add the output directory of the receipt as a resources dir vs copying it as a separate task